setup.py

```diff
@@ -1,15 +1,16 @@
 from setuptools import setup, find_packages
 
 setup(
-    name="xoxzo.cloudpy",
-    version="0.3.2",
+    name="xoxzo_cloudpy",
```

Comment:
@hyejeongpark What the reason of this changes from `.` to under `_` ?

Reply:
@k4ml Oh, I think I read a post and misunderstood it. I changed the name again. Other than, does this look okay?

Reply:
@hyejeongpark have you tested this by installing 2 namespaced packages - xoxzo.cloudpy and xoxzo.something ?